routes/index.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+const os = require("os");
+const fs = require("fs");
+const path = require("path");
+const express = require("express");
+const stream = require("stream");
+const Handlebars = require("handlebars");
+const wkhtmltopdf = require("wkhtmltopdf");
```

Comment:
I was reading about the `wkhtmltopdf` npm package. Sounds like you still need to have the command line tool installed and set on the PATH. If this is true to get this project to work, then let's document it with the other installation/setup documentation.